R/dvhydrograph-render.R

```diff
@@ -20,64 +15,65 @@ createDVHydrographPlot <- function(reportObject){
     'startDate',
     'endDate',
     'isInverted',
-    'timezone'
+    'timezone',
+    'stationId',
+    'stationName',
+    'title'
   )
 
+  #Get Necessary Report Metadata
   if(validateFetchedData(metaData, "metadata", requiredMetadataFields)){
-    #Get Necessary Report Metadata
     timezone <- fetchReportMetadataField(reportObject, 'timezone')
     excludeZeroNegativeFlag <- fetchReportMetadataField(reportObject, 'excludeZeroNegative')
     excludeMinMaxFlag <- fetchReportMetadataField(reportObject, 'excludeMinMax')
     invertedFlag <- fetchReportMetadataField(reportObject, 'isInverted')
+    excludeZeroNegativeFlag <- ifelse(is.null(excludeZeroNegativeFlag), FALSE, TRUE)
+    excludeMinMaxFlag <- ifelse(is.null(excludeMinMaxFlag), FALSE, TRUE)
+    invertedFlag <- ifelse(is.null(invertedFlag), FALSE, TRUE)
```

Comment:
should we use `isEmptyOrBlank` instead of `is.null`? Also, can these be moved so that are directly below their fetcher?